querydsl-mongodb/src/main/java/com/querydsl/mongodb/AbstractMongodbQuery.java

```diff
@@ -411,6 +411,15 @@ public void setReadPreference(ReadPreference readPreference) {
         this.readPreference = readPreference;
     }
 
+    /**
+     * Get the where definition as a DBObject instance
+     *
+     * @return
+     */
+    public DBObject asDBObject() {
+        return createQuery(queryMixin.getMetadata().getWhere());
+    }
+
     @Override
     public String toString() {
         return createQuery(queryMixin.getMetadata().getWhere()).toString();
```

Comment:
Would we be able to reuse the `asDBObject` here in the `toString()`?
I don't know if they will ever diverge, but it reduces repetition right now.